polling_stations/apps/data_importers/management/commands/import_eoni.py

```diff
@@ -39,18 +39,28 @@ def add_arguments(self, parser):
             help="Don't process address, only update stations",
             action="store_true",
         )
+        parser.add_argument(
+            "--cleanup", help="Delete intermediate CSVs", action="store_true"
+        )
 
     def handle(self, *args, **options):
         self.eoni_export_path = Path(options["eoni_csv"])
         self.eoni_data_root = self.eoni_export_path.absolute().parent
         self.addresses_path = self.eoni_data_root / "eoni_address.csv"
         self.uprn_to_council_path = self.eoni_data_root / "eoni_uprn_to_council.csv"
         self.stations_path = self.eoni_data_root / "eoni_stations.csv"
+        self.paths = [
+            self.addresses_path,
+            self.uprn_to_council_path,
+            self.stations_path,
+        ]
```

Comment:
This could be a dict?

```
self.paths = {
    "addresses": self.eoni_data_root / "eoni_address.csv"
    ...etc
}
```
Then 
```
[path.unlink() for path in self.paths.keys() if path.exists()]
```

Reply:
:+1:  [7522603](https://github.com/DemocracyClub/UK-Polling-Stations/pull/4536/commits/752260366348511a3559f2297d1f7aa5700756e6)